server/command.go

```diff
@@ -182,7 +238,7 @@ func showInstallServerHelp(p *Plugin, context *model.CommandArgs, args ...string
 	return &model.CommandResponse{}
 }
 
-func deleteSubscription(p *Plugin, context *model.CommandArgs, args ...string) *model.CommandResponse {
+func deleteSubscription(_ *Plugin, context *model.CommandArgs, args ...string) *model.CommandResponse {
```

Comment:
Can we just remove the unused args?

Reply:
https://github.com/mattermost-community/mattermost-plugin-confluence/pull/126#discussion_r1990085081